### PR TITLE
Tweaks the yuck chem

### PR DESCRIPTION
**Pull request by ATH1909**
## About The Pull Request

This PR makes the following alteration to the yuck chem:
The yuck chem (that one that you make by dipping a dead mouse in welding fuel) no longer purges itself when it makes you vomit until your vomiting starts to stun you. This should make the chem much more viable to actually be used, since it'd normally make you vomit maybe once or twice before purging itself from your system completely unless you used really large doses of it. Now, you should usually get at least to the stun vomiting stage before the chem begins to purge itself (which may be a buff or a nerf to the chem, depending on how you look at it).

## Why It's Good For The Game

The yuck chem alteration makes it more viable to use in non-massive, non-tiny doses, which I think is appropriate for a "ghetto chem".

## Changelog
:cl: ATHATH
balance: The "yuck" chem now only starts to purge itself once you start stun-vomiting.
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm [hunk 1]
```diff
@@ -180,10 +180,10 @@
 		var/datum/reagent/the_reagent = r
 		if(istype(the_reagent, /datum/reagent/medicine))
 			medibonus += 1
-	M.adjustToxLoss(0.5 * medibonus)
-	M.adjustOrganLoss(ORGAN_SLOT_LUNGS, medibonus)
+	M.adjustToxLoss(0.5 * FLOOR(sqrt(medibonus*2),1))
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm]: Also needs to be -0.5 instead of 0.5

**Comment by ExcessiveUseOfCobblestone** [comment]
wait is this tested? the reason why I have higher organ damages is because they don't DO anything if it's too low since organs passively heal on their own.

The reason why chemists are using it now is most likely because it gives toxin damage per medicine reagent you have (bug) on top of liver damage.

**Comment by ATH1909** [comment]
> wait is this tested? the reason why I have higher organ damages is because they don't DO anything if it's too low since organs passively heal on their own.
> 
> The reason why chemists are using it now is most likely because it gives toxin damage per medicine reagent you have (bug) on top of liver damage. I'll want to fix the bug before saying that people use it as a toxin while it heals.

You mean lung damage, not liver damage, right?

Lungs heal at a rate of 0.1 points per tick. I could bump the lung damage rate up to 2 points or even 3 points, if you'd like me to.

**Comment by ExcessiveUseOfCobblestone** [comment]
jesus they have different healing rates.

Just bump it down to 0.1 but keep the medicine amplifier.

I want multiver/palletta to be the more nuclear option for toxin since syriniver (thotlizid) and fiziver handle the more tamer situations.

**Comment by ATH1909** [comment]
> jesus they have different healing rates.
> 
> Just bump it down to 0.1 but keep the medicine amplifier.
> 
> I want multiver/palletta to be the more nuclear option for toxin since syriniver (thotlizid) and fiziver handle the more tamer situations.

Wait, I was slightly wrong: lungs heal 0.1-0.5 health points per tick, depending on how satiated you are.

I don't really see why we need to punish people for going with multiver's gimmick by giving them the same amount of lung damage as the people who don't (assuming that they eventually end up healing the same amount of toxin damage with multiver).

**Comment by ATH1909** [comment]
I would also like to point out that damage to your lungs doesn't really do much until the lungs hit 100 damage points. At the high damage (45 points) threshold, you'll breathe in twice as often, but that's pretty much it until you hit 100 lung damage and stop breathing altogether.

**Comment by Akrilla** [comment]
ghetto chem 👍

**Comment by carlarctg** [comment]
'multiver' isn't exactly the scariest chemical name, and making it have the exact color and spawn in the same places as charcoal doesn't seem like such a good idea to me if it's the 'nuclear option' instead of charcoal's easy fixes

kinda not in scope of this pr but don't know where to put it

**Comment by ExcessiveUseOfCobblestone** [comment]
it replaced charcoal because the tox chem kit already had thializid.

**Comment by ExcessiveUseOfCobblestone** [comment]
> I don't really see why we need to punish people for going with multiver's gimmick by giving them the same amount of lung damage as the people who don't (assuming that they eventually end up healing the same amount of toxin damage with multiver).

It's not a "punishment", it's a way to amplify the medicine. This is also why it does lung damage, the organ that doesn't have too big of an "oh shit" midway effect.

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]
just the yuck changes please, I want to see the other meds in action more.

**Pull request merged**